[2025-02-13 22:03:34,428: INFO: main: logging is implemented]
[2025-02-13 22:03:52,211: INFO: main: logging is implemented]
[2025-02-14 16:17:23,454: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:17:23,458: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:20:22,607: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:20:22,614: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:20:22,615: INFO: common: created directory at: artifacts]
[2025-02-14 16:20:22,616: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:21:53,434: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:21:53,434: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:21:53,437: INFO: common: created directory at: artifacts]
[2025-02-14 16:21:53,438: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:32:33,539: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:32:33,539: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:32:33,542: INFO: common: created directory at: artifacts]
[2025-02-14 16:32:33,543: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:33:41,604: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:33:41,604: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:33:41,604: INFO: common: created directory at: artifacts]
[2025-02-14 16:33:41,604: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:34:41,880: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:34:41,880: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:34:41,886: INFO: common: created directory at: artifacts]
[2025-02-14 16:34:41,887: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:35:27,050: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:35:27,051: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:35:27,052: INFO: common: created directory at: artifacts]
[2025-02-14 16:35:27,052: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:36:22,348: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:36:22,349: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:36:22,350: INFO: common: created directory at: artifacts]
[2025-02-14 16:36:22,351: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:36:41,871: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:36:41,871: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:36:41,874: INFO: common: created directory at: artifacts]
[2025-02-14 16:36:41,876: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:36:55,475: INFO: 3347790719: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: A60D:216E99:3C03A5:40A7CD:67AF38DF
Accept-Ranges: bytes
Date: Fri, 14 Feb 2025 12:36:55 GMT
Via: 1.1 varnish
X-Served-By: cache-fjr990021-FJR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1739536609.464480,VS0,VE6406
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: aebcba35cb118a3cc4d6e9c1af662d02ac304839
Expires: Fri, 14 Feb 2025 12:41:55 GMT
Source-Age: 6

]
[2025-02-14 16:39:41,050: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 16:39:41,052: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 16:39:41,053: INFO: common: created directory at: artifacts]
[2025-02-14 16:39:41,054: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 16:39:46,632: INFO: 1916786552: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: A60D:216E99:3C03A5:40A7CD:67AF38DF
Accept-Ranges: bytes
Date: Fri, 14 Feb 2025 12:39:47 GMT
Via: 1.1 varnish
X-Served-By: cache-fjr990025-FJR
X-Cache: HIT
X-Cache-Hits: 1
X-Timer: S1739536787.045728,VS0,VE0
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: d8dc3baa78058ef81266c8e57e0d615735987f6f
Expires: Fri, 14 Feb 2025 12:44:47 GMT
Source-Age: 177

]
[2025-02-14 18:12:41,512: INFO: main: logging is implemented]
[2025-02-14 18:12:41,512: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-14 18:12:41,528: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 18:12:41,537: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 18:12:41,538: INFO: common: created directory at: artifacts]
[2025-02-14 18:12:41,539: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-14 18:12:42,942: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 7D5F:15F9A4:10662E:25E172:67AF4F5A
Accept-Ranges: bytes
Date: Fri, 14 Feb 2025 14:12:43 GMT
Via: 1.1 varnish
X-Served-By: cache-fjr990029-FJR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1739542363.796677,VS0,VE624
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: edd9e9e9cc267c5e9fc3ac71b303837f3dded553
Expires: Fri, 14 Feb 2025 14:17:43 GMT
Source-Age: 0

]
[2025-02-14 18:12:43,049: INFO: main: stage {STAGE_NAME} completed]
[2025-02-14 19:21:04,860: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-14 19:54:58,111: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 19:54:58,133: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 19:54:58,135: INFO: common: created directory at: artifacts]
[2025-02-14 19:54:58,138: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-14 19:56:00,408: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-14 19:56:00,410: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-14 19:56:00,411: INFO: common: created directory at: artifacts]
[2025-02-14 19:56:00,412: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 13:03:12,171: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 13:03:13,101: INFO: main: logging is implemented]
[2025-02-15 13:03:13,101: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:03:13,112: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:03:13,125: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:03:13,127: INFO: common: created directory at: artifacts]
[2025-02-15 13:03:13,127: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 13:03:13,127: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 13:03:13,240: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 13:03:13,240: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:03:13,241: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:03:13,241: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:03:13,241: INFO: common: created directory at: artifacts]
[2025-02-15 13:03:13,241: ERROR: main: 'function' object has no attribute 'tokenizer_name']
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 35, in <module>
    data_processing_pipeline.initiate_data_transformation()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_data_processing_pipeline.py", line 16, in initiate_data_transformation
    data_ingestion = DataTransformation(config=data_transformation_config)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\data_transformation.py", line 18, in __init__
    self.tokenizer = AutoTokenizer.from_pretrained(config.tokenizer_name)
AttributeError: 'function' object has no attribute 'tokenizer_name'
[2025-02-15 13:04:01,103: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 13:04:01,246: INFO: main: logging is implemented]
[2025-02-15 13:04:01,246: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:04:01,246: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:04:01,246: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:04:01,246: INFO: common: created directory at: artifacts]
[2025-02-15 13:04:01,246: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 13:04:01,246: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 13:04:01,355: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 13:04:01,355: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:04:01,355: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:04:01,355: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:04:01,355: INFO: common: created directory at: artifacts]
[2025-02-15 13:04:01,355: ERROR: main: 'function' object has no attribute 'tokenizer_name']
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 35, in <module>
    data_processing_pipeline.initiate_data_transformation()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_data_processing_pipeline.py", line 16, in initiate_data_transformation
    data_ingestion = DataTransformation(config=data_transformation_config)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\data_transformation.py", line 18, in __init__
    self.tokenizer = AutoTokenizer.from_pretrained(self.config.tokenizer_name)
AttributeError: 'function' object has no attribute 'tokenizer_name'
[2025-02-15 13:06:51,544: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 13:06:51,704: INFO: main: logging is implemented]
[2025-02-15 13:06:51,704: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:06:51,704: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:06:51,704: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:06:51,704: INFO: common: created directory at: artifacts]
[2025-02-15 13:06:51,704: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 13:06:51,704: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 13:06:51,802: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 13:06:51,802: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:06:51,802: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:06:51,802: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:06:51,802: INFO: common: created directory at: artifacts]
[2025-02-15 13:06:51,802: ERROR: main: 'function' object has no attribute 'tokenizer_name']
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 35, in <module>
    data_processing_pipeline.initiate_data_transformation()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_data_processing_pipeline.py", line 16, in initiate_data_transformation
    data_ingestion = DataTransformation(config=data_transformation_config)
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\data_transformation.py", line 18, in __init__
    self.tokenizer = AutoTokenizer.from_pretrained(self.config.tokenizer_name)
AttributeError: 'function' object has no attribute 'tokenizer_name'
[2025-02-15 13:09:34,053: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 13:09:34,211: INFO: main: logging is implemented]
[2025-02-15 13:09:34,211: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:09:34,211: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:09:34,211: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:09:34,211: INFO: common: created directory at: artifacts]
[2025-02-15 13:09:34,211: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 13:09:34,211: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 13:09:34,305: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 13:09:34,305: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 13:09:34,305: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 13:09:34,305: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 13:09:34,305: INFO: common: created directory at: artifacts]
[2025-02-15 13:09:34,305: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 13:09:37,105: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 14:51:49,687: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 16:44:05,594: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 16:44:05,602: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 16:44:05,603: INFO: common: created directory at: artifacts]
[2025-02-15 16:44:05,604: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 16:46:04,140: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 16:46:04,142: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 16:46:04,144: INFO: common: created directory at: artifacts]
[2025-02-15 16:46:04,145: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:20:58,539: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:20:58,541: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:20:58,542: INFO: common: created directory at: artifacts]
[2025-02-15 17:20:58,543: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:22:07,113: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:22:07,117: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:22:07,118: INFO: common: created directory at: artifacts]
[2025-02-15 17:22:07,120: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:23:06,950: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:23:06,952: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:23:06,953: INFO: common: created directory at: artifacts]
[2025-02-15 17:23:06,954: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:30:07,674: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 17:30:07,986: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:30:07,986: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:30:07,986: INFO: common: created directory at: artifacts]
[2025-02-15 17:30:07,986: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:31:38,234: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 17:31:38,436: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:31:38,438: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:31:38,439: INFO: common: created directory at: artifacts]
[2025-02-15 17:31:38,440: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:32:43,889: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 17:32:44,096: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:32:44,101: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:32:44,102: INFO: common: created directory at: artifacts]
[2025-02-15 17:32:44,102: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:36:29,077: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 17:36:29,274: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:36:29,275: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:36:29,276: INFO: common: created directory at: artifacts]
[2025-02-15 17:36:29,276: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:42:25,525: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:42:25,525: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:42:25,528: INFO: common: created directory at: artifacts]
[2025-02-15 17:42:25,529: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:44:21,769: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:44:21,771: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:44:21,773: INFO: common: created directory at: artifacts]
[2025-02-15 17:44:21,773: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:49:20,306: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 17:49:20,577: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:49:20,584: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:49:20,584: INFO: common: created directory at: artifacts]
[2025-02-15 17:49:20,584: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 17:54:36,200: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 17:54:36,207: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 17:54:36,209: INFO: common: created directory at: artifacts]
[2025-02-15 17:54:36,209: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 18:00:39,931: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 18:00:39,932: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 18:00:39,934: INFO: common: created directory at: artifacts]
[2025-02-15 18:00:39,934: INFO: common: created directory at: artifacts/model_training]
[2025-02-15 19:11:59,886: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 19:12:00,329: INFO: main: logging is implemented]
[2025-02-15 19:12:00,331: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 19:12:00,331: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 19:12:00,335: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 19:12:00,335: INFO: common: created directory at: artifacts]
[2025-02-15 19:12:00,337: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 19:12:00,337: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 19:12:00,406: ERROR: main: [Errno 22] Invalid argument: 'artifacts\\data_ingestion\\samsum_dataset\\test\\data-00000-of-00001.arrow']
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 18, in <module>
    data_ingestion_pipeline.initiate_data_ingestion()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_data_ingestion_pipeline.py", line 18, in initiate_data_ingestion
    data_ingestion.extract_zip_file()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\data_ingestion.py", line 42, in extract_zip_file
    zip_ref.extractall(unzip_path)
  File "C:\Users\amitt\AppData\Local\Programs\Python\Python310\lib\zipfile.py", line 1641, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\amitt\AppData\Local\Programs\Python\Python310\lib\zipfile.py", line 1695, in _extract_member
    open(targetpath, "wb") as target:
OSError: [Errno 22] Invalid argument: 'artifacts\\data_ingestion\\samsum_dataset\\test\\data-00000-of-00001.arrow'
[2025-02-15 19:22:12,694: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 19:22:12,993: INFO: main: logging is implemented]
[2025-02-15 19:22:12,993: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 19:22:12,993: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 19:22:12,993: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 19:22:12,993: INFO: common: created directory at: artifacts]
[2025-02-15 19:22:12,993: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 19:22:13,009: INFO: data_ingestion: file already exists artifacts\data_ingestion\data.zip]
[2025-02-15 19:22:13,058: ERROR: main: [Errno 22] Invalid argument: 'artifacts\\data_ingestion\\samsum_dataset\\test\\data-00000-of-00001.arrow']
Traceback (most recent call last):
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\main.py", line 18, in <module>
    data_ingestion_pipeline.initiate_data_ingestion()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\pipeline\stage_data_ingestion_pipeline.py", line 18, in initiate_data_ingestion
    data_ingestion.extract_zip_file()
  File "C:\Users\amitt\Documents\ML-OPS\text-summarization\src\textsummarizer\components\data_ingestion.py", line 42, in extract_zip_file
    zip_ref.extractall(unzip_path)
  File "C:\Users\amitt\AppData\Local\Programs\Python\Python310\lib\zipfile.py", line 1641, in extractall
    self._extract_member(zipinfo, path, pwd)
  File "C:\Users\amitt\AppData\Local\Programs\Python\Python310\lib\zipfile.py", line 1695, in _extract_member
    open(targetpath, "wb") as target:
OSError: [Errno 22] Invalid argument: 'artifacts\\data_ingestion\\samsum_dataset\\test\\data-00000-of-00001.arrow'
[2025-02-15 19:23:11,260: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 19:23:11,541: INFO: main: logging is implemented]
[2025-02-15 19:23:11,541: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 19:23:11,541: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 19:23:11,541: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 19:23:11,541: INFO: common: created directory at: artifacts]
[2025-02-15 19:23:11,541: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 19:23:14,261: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: E443:3403CA:793D23:80E91F:67B0B160
Accept-Ranges: bytes
Date: Sat, 15 Feb 2025 15:23:14 GMT
Via: 1.1 varnish
X-Served-By: cache-fjr990023-FJR
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1739632992.357555,VS0,VE1704
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 504ab52a68fee4b87c7e908a78acdf170fb27fac
Expires: Sat, 15 Feb 2025 15:28:14 GMT
Source-Age: 1

]
[2025-02-15 19:23:14,367: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 19:23:14,367: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 19:23:14,367: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 19:23:14,367: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 19:23:14,367: INFO: common: created directory at: artifacts]
[2025-02-15 19:23:14,367: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 19:23:17,296: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 19:23:17,296: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 19:23:17,297: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 19:25:21,643: INFO: config: PyTorch version 2.6.0 available.]
[2025-02-15 19:25:21,935: INFO: main: logging is implemented]
[2025-02-15 19:25:21,935: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 19:25:21,935: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 19:25:21,935: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 19:25:21,935: INFO: common: created directory at: artifacts]
[2025-02-15 19:25:21,935: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 19:25:22,311: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 7903594
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "dbc016a060da18070593b83afff580c9b300f0b6ea4147a7988433e04df246ca"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: E443:3403CA:793D23:80E91F:67B0B160
Accept-Ranges: bytes
Date: Sat, 15 Feb 2025 15:25:22 GMT
Via: 1.1 varnish
X-Served-By: cache-fjr990033-FJR
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1739633122.289859,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 6a3a3544e17f50febdacc4941ea0082a12331cf3
Expires: Sat, 15 Feb 2025 15:30:22 GMT
Source-Age: 129

]
[2025-02-15 19:25:22,405: INFO: main: stage { STAGE_NAME } completed]
[2025-02-15 19:25:22,405: INFO: main: stage {STAGE_NAME} initiated]
[2025-02-15 19:25:22,405: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 19:25:22,405: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 19:25:22,405: INFO: common: created directory at: artifacts]
[2025-02-15 19:25:22,421: INFO: common: created directory at: artifacts/data_transformation]
[2025-02-15 19:25:25,014: INFO: main: stage {STAGE_NAME} completed]
[2025-02-15 19:25:25,015: INFO: main: stage { STAGE_NAME } initiated]
[2025-02-15 19:25:25,016: INFO: main: stage { STAGE_NAME } completed]
